src/providers/ldap/sdap_online_check.c

```diff
@@ -223,11 +227,44 @@ static void sdap_online_check_handler_done(struct tevent_req *subreq)
     ret = sdap_online_check_recv(subreq);
     talloc_zfree(subreq);
 
+    if (ret == EOK) {
+        subreq = dp_req_send(state->id_ctx->be, state->id_ctx->be->provider,
```

Comment:
I don't insist but maybe it is worth adding a comment here why we are performing this subdomain check here?

Reply:
I added a comment.